modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -9476,7 +9476,7 @@
                   "errors": [
                     {
                       "title": "Unprocessable Entity",
-                      "detail": "Unable to locate Veteran's File Number in Master Person Index (MPI). Please submit an issue at ask.va.gov or call 1-800-MyVA411 (800-698-2411) for assistance.",
+                      "detail": "The SSN provided does not match Master Person Index (MPI). Please correct the SSN or submit an issue at ask.va.gov or call 1-800-MyVA411 (800-698-2411) for assistance.",
```

Comment:
we can ignore this for this PR to cut down on LoC. Although it's not much. I also think this is in another PR so it can be resolved there.